docs/index.md

```diff
@@ -5,7 +5,8 @@ layout: docs
 
 Local Kubernetes development with no stress.
 
-[Tilt]({{site.landingurl}}) helps you develop your microservices locally.
+[Tilt]({{site.landingurl}}) helps you develop your microservices locally,
+especially if [you deploy to Kubernetes (or are planning to)](https://blog.tilt.dev/2020/02/12/local-dev.html).
```

Comment:
i think you can remove the parenthetical

Reply:
Done. (Assuming you mean remove the parenthesis symbols, and not the content in them.